store/postgres/migrations/2024-08-14-205601_store_synced_at_block/up.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE subgraphs.subgraph_deployment ADD COLUMN synced_at_block_number NUMERIC;
```

Comment:
It would be better to make that an `int4`, too. The fact that `latest_ethereum_block_number` is `numeric` is more a historical accident than anything we should continue doing for other places where we store block numbers. In an ideal world, we'd change al the columns that store block numbers and use `numeric` to `int4` but that might be a bit delicate and nobody's done that, obviously.